apps/offlineevents/views.py

```diff
@@ -10,3 +19,102 @@ class OfflineEventDetailView(module_views.ItemDetailView):
     @property
     def project(self):
         return self.get_object().project
+
+
+class OfflineEventMgmtView(DashboardBaseMixin,
```

Comment:
I am not a big fan of naming the methods Mgmt/managment, as those are very special views that have to be initially routed via the quirky lookup found due to a PhaseContent management_view property. 
Offline events are different as they will be available for every project regardless of the blueprint/phases.
Also every  view of the offline events settings are routed via it's own urls.py.

I suggest to either remove the Mgmt/management and the manage route altogether or replace it by something called dashboard.

For the routes i would prefer `dashboard/projects/{slug}/offlineevents` analogous to 
 `dashboard/projects/{slug}/moderators` and the management  start point `dashboard/projects/{slug}/manage`-
Unfortunately this would be a little messay, as this route would have to be defined in the context of the dashboard app :(